prefs.js

```diff
@@ -80,7 +78,15 @@ const pretty_names = {
     'preset-resize-27'  : 'Preset resize 27',
     'preset-resize-28'  : 'Preset resize 28',
     'preset-resize-29'  : 'Preset resize 29',
-    'preset-resize-30'  : 'Preset resize 30'
+    'preset-resize-30'  : 'Preset resize 30',
+    'move-left-vi'      : 'vi-style Resize horizontal narrower',
```

Comment:
This needs a better description because currently it is the same as the actual resize action, while this is supposed to be a move action.

Reply:
You are correct